osis-app/src/main/java/com/scality/osis/resource/OsisController.java

```diff
@@ -1,15 +1,11 @@
-/**
```

Comment:
You're definitely breaching the original license by removing copyright headers etc.

Reply:
Fixed this and added copyright headers along with `Scality`.